go/vt/vtgate/planbuilder/symtab.go

```diff
@@ -63,10 +63,7 @@ func newSymtab(vschema VSchema) *symtab {
 }
 
 // AddAlias adds a table alias to symtab.
-func (st *symtab) AddAlias(alias, astName sqlparser.TableIdent, table *vindexes.Table, rb *route) error {
-	if found := st.findTable(alias); found != nil {
```

Comment:
Duplicate symbols are disallowed at a higher level (in the merge function). This was basically unreachable code. In fact, this function could very well be folded into the create symbol table call, which is the only context where it's referenced.